src/db/collections/protocol_update.rs

```diff
@@ -92,17 +92,17 @@ impl ProtocolUpdateCollection {
     }
 
     /// Add the protocol parameters to the list if the protocol parameters have changed.
-    pub async fn update_latest_protocol_parameters(
+    pub async fn upsert_protocol_parameters(
```

Comment:
Should `insert_protocol_parameters` become private now as both cases (insertion, updating) are handled by this method, and the user can't keep track of whether it was inserted already anyway?